src/main/java/com/kainos/ea/backend/models/User.java

```diff
@@ -9,26 +7,21 @@
 @Entity
 @Table(name = "User")
 public class User {
-
     @Id
     @Column(name = "username")
     private String username;
-
     @Column(name = "password")
     private String password;
-
     @Column(name = "isAdmin")
     private boolean isAdmin;
 
     public User() {
     }
-
```

Comment:
Arguably this is my fault because I have now scared you off using white space all together. However, this is now too far the other way 😂  1x line of white space is expected between functions. The same for between your package definition and imports. Again, refer to the auto-formatting in your IDE for some guidance here.

Reply:
Okay, added back the good whitespaces :D